src/barchart_split/BarChartSplit.js

```diff
@@ -0,0 +1,228 @@
+import Base from "../commons/base";
+import { select, selectAll } from 'd3-selection';
+import { min, max } from 'd3-array';
+import { nest } from 'd3-collection';
+import { timeMonth, timeYear } from "d3-time";
+import { timeFormat } from "d3-time-format";
+import { scaleLinear, scaleBand, scaleOrdinal } from 'd3-scale';
+import { axisBottom, axisLeft } from 'd3-axis';
+import "d3-transition";
+import "./BarChartSplit.css"
+
+export default class BarChartSplit extends Base {
+  constructor(container, data, options = {}) {
+    super(container, data, options)
+
+    // main properties to display
+    this.xAxisProp = options.x;
+    this.yAxisProp = options.y;
+    this.countProp = options.count;
+    this.height = options.height || 600
+    this.arrayOfScales = [];
+    this.arrayOfXaxisProp = [];
```

Comment:
You may use just a _plural_ word to name arrays, that is:
```suggestion
    this.scales = [];
    this.xAxisProps = [];
```

... what leads me to confusion: which is the difference between this.xAxisProp and this.xAxisProps? This last one should have a slightly significative name